lldb/test/API/lang/swift/cxx_interop/forward/verbose_trap/TestSwiftForwardInteropVerboseTrap.py

```diff
@@ -9,6 +9,7 @@
 class TestSwiftForwardInteropVerboseTrap(TestBase):
 
     @swiftTest
+    @expectedFailureAll()
```

Comment:
Ideally we'd track this with a Radar and add a `bugnumber="rdar://1234` argument to the decorator.